modules/builds-basic-edit-buildconfig.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * builds/basic-build-operations.adoc
+
+[id="builds-basic-edit-buildconfig_{context}"]
+= Editing a BuildConfig
+
+You can use the *Edit BuildConfig* option in the *Builds* view of the *Developer* perspective to edit your build configurations. You can edit a `BuildConfig` using the following options:
+
+* *Form view*: This allows you to edit and configure your `BuildConfig` using the standard form fields and checkboxes.
+* *YAML view*: This allows you to edit and configure your `BuildConfig` manually with full control over the operations.
+
+You can also switch between the *Form view* and *YAML view* without losing any data. The data in the *Form view* is transferred to the *YAML view* and vice versa.
+
+Use the following instructions to edit the `BuildConfig` using the Form view of the Developer perspective:
+
+* In the Builds view of the Developer perspective, click on the menu {kebab} to see *Edit BuildConfig*.
+* Click on *Edit BuildConfig* to find the *Form view* configure option.
+* In the Git section, enter the Git repository URL for the codebase you want to use to create an application. The URL is then validated.
+* Optional: You can click *Show Advanced Git Options* to add details such as:
+** *Git Reference* to point to code in a specific branch, tag, or commit to be used to build the application.
+** *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
+** *Source Secret* to create a Secret Name with credentials for pulling your source code from a private repository.
+* In the *Build from* section, click on the option that you would like to build from. You can use the following options:
+** *Image Stream tag*: The Image Stream tag is used to reference an image for a given image stream and tag. You will need to enter the project, image stream and tag to refer to where you would like to build from and push to.
+** *Image Stream image*: The Image Stream image is used to reference an image for a given image stream and image name. You will need to enter the image stream image from where you would like to build. You will also need to enter the project, image stream and the tag to refer to where you would like to push to.
```

Comment:
```suggestion
** *Image Stream image* references an image for a given image stream and image name. You will need to enter the image stream image from where you would like to build. You will also need to enter the project, image stream and the tag to refer to where you would like to push to.
```
Avoid repetition.
Avoid future tense.
Use imperative verbs to give instructions.